features/distributed/kv-store/blob/blob-itests/src/test/java/org/opennms/features/distributed/blob/cassandra/CassandraBlobStoreIT.java

```diff
@@ -28,171 +28,50 @@
 
 package org.opennms.features.distributed.blob.cassandra;
 
-import static org.junit.Assert.fail;
-
 import java.io.IOException;
 import java.net.InetSocketAddress;
-import java.util.Collections;
-import java.util.List;
-import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
 
-import org.cassandraunit.CassandraCQLUnit;
-import org.cassandraunit.dataset.CQLDataSet;
-import org.junit.Rule;
-import org.junit.Test;
+import org.junit.ClassRule;
 import org.opennms.features.distributed.blob.BaseBlobStoreIT;
 import org.opennms.features.distributed.cassandra.api.CassandraSession;
-import org.opennms.features.distributed.kvstore.api.BlobStore;
 import org.opennms.features.distributed.kvstore.api.SerializingBlobStore;
 import org.opennms.features.distributed.kvstore.blob.cassandra.CassandraBlobStore;
+import org.opennms.newts.cassandra.NewtsInstance;
 import org.opennms.newts.cassandra.SchemaManager;
 
-import com.datastax.driver.core.PreparedStatement;
-import com.datastax.driver.core.RegularStatement;
-import com.datastax.driver.core.ResultSet;
-import com.datastax.driver.core.ResultSetFuture;
-import com.datastax.driver.core.Session;
-import com.datastax.driver.core.Statement;
-import com.datastax.driver.core.exceptions.DriverException;
+import org.opennms.features.distributed.cassandra.impl.NewtsCassandraSessionFactory;
 
 public class CassandraBlobStoreIT extends BaseBlobStoreIT {
-    private static final String KEYSPACE = "opennms";
-
-    private final CQLDataSet testSet = new CQLDataSet() {
-        @Override
-        public List<String> getCQLStatements() {
-            return Collections.emptyList();
-        }
-
-        @Override
-        public String getKeyspaceName() {
-            return null;
-        }
-
-        @Override
-        public boolean isKeyspaceCreation() {
-            return false;
-        }
 
-        @Override
-        public boolean isKeyspaceDeletion() {
-            return false;
-        }
-    };
-
-    @Rule
-    public CassandraCQLUnit cassandraUnit = new CassandraCQLUnit(testSet);
+    @ClassRule
+    public static NewtsInstance newtsInstance = new NewtsInstance();
```

Comment:
This uses TestContainers, which means we now rely on Docker for ITs outside of the `smoke-tests`.